holoviews/plotting/bokeh/element.py

```diff
@@ -524,6 +524,8 @@ def _axis_props(self, plots, subplots, element, ranges, pos, *, dim=None,
             else not util.isfinite(el) for el in [v0, v1]
         ):
             dim_range = range_type()
+        elif issubclass(range_type, FactorRange):
+            dim_range = range_type(name=dim.name if dim else None)
```

Comment:
@philippjfr, do you have any fear about this change?  By not setting start and end on `FactorRange`.

Reply:
None, whatsoever. Looks fine.